tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -630,6 +630,8 @@ def _screenshot(self, protocol, url, timeout):
         self.protocol.base.execute_script(self.wait_script, True)
 
         screenshot = self.protocol.webdriver.screenshot()
+        if screenshot is None:
+            raise ValueErrori('screenshot is None')
```

Comment:
You'll need to fix the typo to make CI pass :)

Reply:
Thanks